app/models/season.rb

```diff
@@ -13,6 +13,10 @@ def succ
       find_or_create_by(name: (Date.today.year+1).to_s)
     end
 
+    def prev
+      find_or_create_by(name: (Date.today.year-1).to_s)
+    end
```

Comment:
I tried that first, and didn't find a neat way to do that. 
In the teams factory, there is a trait :current_season, which calls the class method. I thought it will get very messy if I have to add a :last_season trait there, if it is one without a call to the class method. That's why I chose the option with a newly added class method.
The other solution I explored is to add a :last_season factory to the Season factory, but that would mean (AFAIK) that I would have to make associations from team factory to season factory. Apart from that I don't want to get into that, I think that would mess up all the RSpecs as well. 
Shall I go for the 'messy' :last_season trait in the team factory then?